package.json

```diff
@@ -33,7 +33,9 @@
       "build/actions-oclif",
       "build/auth/pages/*.ejs",
       "build/hooks",
-      "node_modules/resin-discoverable-services/services/**/*"
+      "node_modules/resin-discoverable-services/services/**/*",
+      "node_modules/windosu/*.bat",
+      "node_modules/windosu/*.cmd"
```

Comment:
This is one of the fixes required to get `windosu` to work with standalone zip packages. Even with the changes in this PR, I found that `os intialize` still doesn't work with standalone zip package on Windows specifically. It works with a npm install on Windows, it works with standazone zip package on Linux / Mac, but it doesn't work with standalone zip package on Windows. Presently clueless as to why. I'm inclined to create a CLI issue instead of spending more time on this.

Reply:
As long as we're moving forwards, I'd be ok with that.

Reply:
I've managed to identify why `os initialize` was not working in the combination of standalone zip package and Windows, and fixed it. It was because `windosu` would spawn a child process using `process.execPath` as the path of the Node.js interpreter. However, in a standalone zip package, Node.js is built-in in the `balena.exe` binary and `process.execPath` is actually `balena.exe`. When a standalone zip package is being used, the OS may not have a separate Node.js installation at all. To solve this, I came up with a new internal command, `pkgExec`, that `require`s the module given as argument, and I patched `windosu` to use it when a standalone zip package is detected. This was done in a separate commit.